### PR TITLE
Documentation: add warning to "creating an addon"

### PR DESCRIPTION
**Pull request by WorldLanguages**
Links to the contributing guidelines, in case the reader intends to submit their addon instead of creating it for personal use or for other purposes.

The rest of the article should be re-written at some point.

### CODE REVIEW

**Comment by WorldLanguages** [comment]
If you have any suggestions on how this should be phrased, please tell me. I'll merge this in a few days,

File: content/docs/develop/getting-started/creating-an-addon.md [hunk 1]
```diff
@@ -7,6 +7,15 @@ aliases:
 Required software: text editor, Chrome.  
 If possible, disable the Scratch Addons extension you downloaded from the store before proceeding to avoid issues.
 
+
+{{< admonition error >}}
```
Comment (Hans5958) [inline content/docs/develop/getting-started/creating-an-addon.md:11]: What's with the error? Why not info?

```suggestion
{{< admonition info>}}
```

Comment (WorldLanguages) [reply content/docs/develop/getting-started/creating-an-addon.md:11]: Sure. Just wanted to use red for some reason.

Comment (Hans5958) [reply content/docs/develop/getting-started/creating-an-addon.md]: Part of me wants to give a meaning of these colors.

**Pull request merged**